android/capacitor/src/main/java/com/getcapacitor/BridgeWebViewClient.java

```diff
@@ -92,4 +93,9 @@ public void onReceivedHttpError(WebView view, WebResourceRequest request, WebRes
             view.loadUrl(errorPath);
         }
     }
+
+    @Override
+    public boolean onRenderProcessGone(final WebView view, RenderProcessGoneDetail detail) {
+        return bridge.onRenderProcessGone(view, detail);
```

Comment:
The webview is passed down as the `onRenderProcessGone` method is called for **any** render crashes within the app (including a pop-up tab, for example).